Dockerfile

```
@@ -0,0 +1,29 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS build
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt ./
+
+RUN pip install --upgrade pip && \
+    pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN python manage.py migrate
```

You can use one `copy` to copy one file, The running of migration should be on run stage 
```suggestion
COPY . .
```